packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -16,9 +16,16 @@
 	}
 }
 
+.edit-post-visual-editor,
+.edit-post-visual-editor > .block-editor__typewriter,
+.edit-post-visual-editor > .block-editor__typewriter > .block-editor-writing-flow,
+.edit-post-visual-editor > .block-editor__typewriter > .block-editor-writing-flow > .block-editor-writing-flow__click-redirect {
+	height: 100%;
```

Comment:
I see that `.block-editor-writing-flow__click-redirect` is smaller than `.block-editor-writing-flow`, taking into account the rest of the content, even though it's set to 100%. How does this work?

Reply:
> I see that `.block-editor-writing-flow__click-redirect` is smaller than `.block-editor-writing-flow`, taking into account the rest of the content, even though it's set to 100%. How does this work?

If I understand correctly:

`.block-editor-writing-flow` is a flex container. I recall that in the past, it was very specifically implemented that the click redirect have a base height to occupy the full space, shrinking if necessary. I think those specific shrink styles might have since been removed, but apparently it still works? I might want to do a round of cross-browser testing, and perhaps add back that attribute just to be explicit.